projects/storefrontlib/src/cms-components/product/product-list/product-facet-navigation/facet-list/facet-list.component.ts

```diff
@@ -21,10 +23,21 @@ import { FacetService } from '../services/facet.service';
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class FacetListComponent {
+  private _isDialog: boolean;
   /**
    * Indicates that the facet navigation is rendered in dialog.
    */
-  @Input() isDialog: boolean;
+  @Input()
+  set isDialog(value: boolean) {
+    this._isDialog = value;
+    if (value) {
+      this.renderer.addClass(document.body, 'modal-open');
```

Comment:
Should we also add `aria-modal="true"`?

Reply:
I don't mind leaving this to the next round of accessibility that is planned post 2.0